mftf/2.3/merging.md

```diff
@@ -8,7 +8,14 @@ redirect_from: /guides/v2.3/magento-functional-testing-framework/2.3/merging.htm
 _This topic was updated due to the {{page.mftf-release}} MFTF release._
 {: style="text-align: right"}
 
-The MFTF allows you to merge test components defined in XML files, such as [tests], [pages][page], [sections], and [data].
+The MFTF allows you to merge test components defined in XML files, such as:
+
+- [tests]
```

Comment:
Are these called "tests", "sections", etc. in the code? If so they should be in code formatting, encased with backticks. If not, consider capitalizing them since they are the first word in a bullet point.

Reply:
Fixed.